core/rawdb/accessors_chain.go

```diff
@@ -1137,3 +1137,71 @@ func ReadHeadBlock(db ethdb.Reader) *types.Block {
 	}
 	return ReadBlock(db, headBlockHash, *headBlockNumber)
 }
+
+// ReadEtxSetRLP retrieves the EtxSet corresponding to a given block, in RLP encoding.
+func ReadEtxSetRLP(db ethdb.Reader, hash common.Hash, number uint64) rlp.RawValue {
+	// First try to look up the data in ancient database. Extra hash
+	// comparison is necessary since ancient database only maintains
+	// the canonical data.
+	data, _ := db.Ancient(freezerEtxSetsTable, number)
```

Comment:
Are we using the freezer table on write ?

Reply:
No, but ancient items get migrated to the freezer table, so we need to check there. This is just a direct copy/paste of `ReadBody()`, so the logic is (hopefully) identical and well tested.

Reply:
logic looks identical. So it should be good.